dttools/src/catalog_query.h

```diff
@@ -20,6 +20,10 @@ Query the global catalog server for server descriptions.
 #define CATALOG_HOST (getenv("CATALOG_HOST") ? getenv("CATALOG_HOST") : CATALOG_HOST_DEFAULT )
 #define CATALOG_PORT (getenv("CATALOG_PORT") ? atoi(getenv("CATALOG_PORT")) : CATALOG_PORT_DEFAULT )
 
+/* Name for the automatically generated ids for records generated by
+ * non-services (e.g. the resource_monitor). */
+#define CATALOG_RECORD_ID_FIELD "cctools_catalog_record_id"
```

Comment:
Suggest just keeping it `record_id` for consistency throughout.

Reply:
I like better your idea of simply calling it 'uuid'. What do you think?